www/src/scss/_theme-moody-blue.scss

```diff
@@ -75,6 +75,11 @@ $highlight-text: #cfd8dc;
     color: $text-base;
   }
 
+  .shortcut-tray { background: darken($bg-main, 10); }
```

Comment:
will create a mixin for all the repetitive code in all the css files during next css cleanup sprint.